cel/io.go

```diff
@@ -120,3 +130,178 @@ func AstToString(a *Ast) (string, error) {
 	info := a.SourceInfo()
 	return parser.Unparse(expr, info)
 }
+
+// RefValueToValue converts between ref.Val and Value.
```

Comment:
nit: mention that the `Value` is the serialized proto form
RefValueToValue serializes a ref.Val to an exprpb.Value proto.

Reply:
Addressed in latest commit